src/react-apps/ux-editor/src/sagas/conditionalRendering/conditionalRenderingSagas.ts

```diff
@@ -107,22 +107,32 @@ export function* checkIfConditionalRulesShouldRun(): SagaIterator {
       const result = (window as any).conditionalRuleHandlerObject[functionToRun](newObj);
       const action = connectionDef.selectedAction;
       // Perform action on the necccessary componenets
-      for (const componentToPerformActionOn in connectionDef.selectedFields) {
+      for (const elementToPerformActionOn in connectionDef.selectedFields) {
         // tslint:disable-next-line:curly
-        if (!componentToPerformActionOn) continue;
-        const comp = formDesignerState.layout.components[connectionDef.selectedFields[componentToPerformActionOn]];
+        if (!elementToPerformActionOn) continue;
+
+        // Either component or container
+        const elementIsComponent = formDesignerState.layout.components[connectionDef.selectedFields[elementToPerformActionOn]] ? true : false;
```

Comment:
elementIsComponent  is kind of useless? 
doesn't this have the same effect:
let element = formDesignerState.layout.components[connectionDef.selectedFields[elementToPerformActionOn]] ?
          formDesignerState.layout.components[connectionDef.selectedFields[elementToPerformActionOn]] :
          formDesignerState.layout.containers[connectionDef.selectedFields[elementToPerformActionOn]];
And since elementIsComponent is only used once its not really necessary in this case